setup.py

```diff
@@ -30,13 +30,17 @@
     install_requires=["PyStemmer>=1.3,<2.0",
                       "bblfsh>=2.2.1,<3.0",
                       "modelforge>=0.7.0,<0.8",
-                      "sourced-engine>=0.5.1,<0.7",
-                      "humanize>=0.5.0",
+                      "sourced-engine>=0.7.0,<0.8",
+                      "humanize>=0.5.0,<0.6",
                       "parquet>=1.2,<2.0",
                       "pygments>=2.2.0,<3.0",
-                      "keras>=2.0,<3.0"] + typing,
+                      "keras>=2.0,<3.0",
+                      "docker>=3.4.0,<4.0",
```

Comment:
We already have docker in `tests-require`

Reply:
Removing then